compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -3173,13 +3338,33 @@ public BLangNode currentAction() {
             return this.actions.get(this.currentState);
         }
 
+        private boolean checkAndSetWorkerMultipleReceiveSource(BLangWorkerMultipleReceive multipleReceive,
```

Comment:
minor suggestion to change `recievedWorker` variable name. `reciev+ed` suggest that receiver has already received. Therefore `receivedWorker.getHasReceived()` feels bit odd.

Reply:
Renamed `recievedWorker` to `receiveField`.